addons/cookoff/XEH_postInit.sqf

```diff
@@ -89,7 +89,7 @@ GVAR(cacheTankDuplicates) = call CBA_fnc_createNamespace;
 
 // blow off turret effect
 ["Tank", "killed", {
-    if ((_this select 0) getVariable [QGVAR(enable),GVAR(enable)]) then {
+    if ((_this select 0) getVariable [QGVAR(enable), GVAR(enable)] in [1, 2, true]) then {
```

Comment:
This is a killed effect. 2 (player crewed only) is treated exactly the same as 1/true (enabled), even though no player may be inside the vehicle. Weird.

Reply:
That's my mistake, will fix.

Reply:
Switched logic to only check if disabled as doesn't matter if it's 1 or 2.